### ISSUE TITLE
models/densenet line193 error

### ISSUE THREAD

**Issue by EricKani** [kenshohara/3D-ResNets-PyTorch#145]
Why there is a Batchnorm2d in 3D convnet...

**Comment by mijung-kim** [comment]
@EricKani Simply changing 2d to 3d worked for me, if you have an error.

**Comment by EricKani** [comment]
thank you～




| |
信工 刘蓬博
邮箱：pengbo18555@163.com
|

签名由 网易邮箱大师 定制

On 05/08/2019 12:32, Mijung KIM wrote:

@EricKani Simply changing 2d to 3d worked for me, if you have an error.
